utils/tagged_integer.hh

```diff
@@ -84,6 +84,15 @@ using tagged_integer = tagged_tagged_integer<struct final, Tag, ValueType>;
 
 } // namespace utils
 
+template <typename Final, typename Tag, std::integral ValueType>
+struct fmt::formatter<utils::tagged_tagged_integer<Final, Tag, ValueType>> {
```

Comment:
might want to `#include <fmt/core.h>`. see https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/5668/consoleFull#639633652dae97f28-980f-4dce-a662-1b9b15a76cc1

Reply:
Sure, fixed. I will push updates soon.

Apologies - I haven't run `dbuild` after seeing that the error does not occur when building natively. However, other translation units that use this header did not compile.

I will run build+tests via `dbuild` before pushing next changes :+1:

Reply:
yeah, neither do i have an explanation for this: `version_generator.cc` only includes `version_generator.hh`, which in turn only includes `tagged_integer.hh`. so i think it should be reproduciable with both building environments.